js/ui/form/components/field_item.js

```diff
@@ -112,8 +112,14 @@ export function renderFieldItem({
         const isToggleControls = toggleControls.includes(editorType);
         const labelAlignment = labelOptions.alignment;
         const isLabelAlignmentLeft = labelAlignment === 'left' || !labelAlignment;
-
-        if(isToggleControls && labelLocation === 'top' && isLabelAlignmentLeft) {
+        const hasNotTemplate = !template;
+        const isLabelOnTop = labelLocation === 'top';
+
+        if(hasNotTemplate
```

Comment:
Let's move it to the next line? It's a preferred syntax according to airbnb: [discussion](https://github.com/airbnb/javascript/issues/1380). As a minimum, it's simpler to comment a line. 

Sorry, of course, formatter should fix this without your attention :(
```suggestion
        if(
            hasNotTemplate
```